src/lib.rs

```diff
@@ -1,8 +1,15 @@
 pub mod auth;
 pub mod response;
-pub mod simple_cache_client;
 
 mod endpoint_resolver;
 mod grpc;
 mod jwt;
+mod simple_cache_client;
 mod utils;
+
+pub use crate::response::error::MomentoError;
+pub use crate::simple_cache_client::{
+    request_meta_data, Fields, IntoBytes, SimpleCacheClient, SimpleCacheClientBuilder,
```

Comment:
I'm not sure if `request_meta_data` is meant to be public since it looks like a helper function.

Reply:
nah, metadata should be an internal helper.

Reply:
Cool. I've removed it from the export and made it private.